setup.json

```diff
@@ -92,7 +92,9 @@
         "aiida_core[atomic_tools]~=1.3",
         "packaging",
         "qe-tools~=2.0rc1",
-        "xmlschema~=1.2"
+        "xmlschema~=1.2",
+        "numpy",
+        "scipy"
```

Comment:
where are we using `scipy`?

Reply:
Whoops, that part has moved into `qe-tools` - removed it now.